src/basics/Basics.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Php version 8.1.9
+ * 
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
+ */
+namespace basics;
+
+/**
+ * Basics Class and implement the BasicsInterface and following methods:
+ * getMinuteQuarter(), isLeapYear(), isSumEqual()
+ * See details below.
+ * 
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
+ */
+class Basics implements BasicsInterface
+{
+    /**
+     * Stores validator object 
+     *
+     * @var mixed _validator - private value for storing validator object 
+     * of class ValidatorInterface, whitch used by all functions to validate
+     * input parameter values for throwing InvalidArgumentException
+     * when the parameter values are out of accepted bounds
+     */
+    private mixed $_validator;
```

Comment:
please use camelCase name convention and the type of it should be BasicsValidatorInterface instead of mixed